src/services/AssetService.ts

```diff
@@ -1,18 +1,25 @@
 // tslint:disable:max-classes-per-file
-import { SearchResult } from '../store';
 import { Endpoints, FlowEditorConfig, ContactProperties, AttributeType } from '../flowTypes';
 import axios, { AxiosResponse } from 'axios';
 import { FlowComponents } from '../store/helpers';
+import { dump } from '../utils';
+
+export interface Asset {
+    id: string;
+    name: string;
+    type: string;
+
+    isNew?: boolean;
+    content?: any;
+}
 
 export class Assets {
     private endpoint: string;
     private localStorage: boolean;
     protected nameProperty: string;
```

Comment:
should we be typing `nameProperty`, `idProperty` and `assetType`? we've used Enums in the past, curious whether you think thats good practice or not

Reply:
agreed, enum makes more sense here